db/migrate/20190507161337_create_dfe_data_tables.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDfEDataTables < ActiveRecord::Migration[5.2]
+  def change
+    create_table :dfe_data_tables, id: :uuid do |t|
+      t.string     :file_name
+      t.belongs_to :admin_user, type: :uuid
```

Comment:
Could we add an explicit `foreign_key` here?

Reply:
Done